tests/msm/mod.rs

```diff
@@ -119,15 +112,15 @@ pub fn input_generator_bls12_377(
     (bases, scalars, msm_result, results)
 }
 
-pub fn precompute_base_bls12_377(base: bls377G1Affine) -> Vec<u8> {
+pub fn precompute_base_bls12_377(base: bls377G1Affine, precompute_factor: u32) -> Vec<u8> {
     let mut bases = vec![];
     let mut current_point = base;
     let x_bytes = current_point.x.into_repr().to_bytes_le();
     let y_bytes = current_point.y.into_repr().to_bytes_le();
     bases.extend_from_slice(&x_bytes);
     bases.extend_from_slice(&y_bytes);
     let two = num_bigint::BigUint::from(2u32);
-    for i in 1..PRECOMPUTE_FACTOR_BLS377 {
+    for i in 1..precompute_factor {
```

Comment:
This won't run for `precompute_factor == PRECOMPUTE_FACTOR_BASE`, we can return early at the beginning of the function if that is the case

Reply:
Yes, idea if precompute factor == 1 we just write point itself and return from function